docs/references.rst

```diff
@@ -112,6 +112,10 @@ References
            doi:`10.1175/1520-0493(1999)127%3C2709%3ATUAMOC%3E2.0.CO;2
            <https://doi.org/10.1175/1520-0493(1999)127%3C2709%3ATUAMOC%3E2.0.CO;2>`_.
 
+.. [Smithsonian1951] Smithsonian Institution and List, Robert J. 1951: `Smithsonian meteorological tables <_static/Smithsonian1951.pdf>`
+            meteorological tables. *Smithsonian Miscellaneous Collections*, **144**,
```

Comment:
Oops, my suggestion didn't remove the meteorological tables from this line, so it says "Smithsonian meteorological tables meteorological tables. Need to clean up and probably wrap the long line.